Source/ACE/Entity/Landblock.cs

```diff
@@ -442,6 +442,99 @@ private void HandleGameAction(QueuedGameAction action, Player player)
                         HandleSoundEvent(obj, soundEffect);
                         break;
                     }
+                case GameActionType.PutItemInContainer:
+                    {
+                        var playerId = new ObjectGuid(action.ObjectId);
+                        var inventoryId = new ObjectGuid(action.SecondaryObjectId);
+                        if (playerId.IsPlayer())
+                        {
+                            Player aPlayer = null;
+                            WorldObject inventoryItem = null;
+
+                            if (worldObjects.ContainsKey(playerId))
+                            {
+                                aPlayer = (Player)worldObjects[playerId];
+                                inventoryItem = worldObjects[inventoryId];                                

```

Comment:
We can't assume inventoryId is in this landblock.  We need something like FindWorldObject which walks this and adjacent landblocks to find it.